semantic_steve/js/src/semantic-steve.ts

```
@@ -0,0 +1,365 @@
+import { buildSkillsRegistry } from "./skill";
+import PItemLoader, { Item as PItem } from "prismarine-item";
+import { Window as PWindow, WindowsExports } from "prismarine-windows";
+import nbt from "prismarine-nbt";
+import { SkillResult, SemanticSteveConfig } from "./types";
+import {
+  InventoryChanges,
+  InventoryChangesDTO,
+  getInventoryChangesDTO,
+} from "./inventory-changes";
+import { GenericSkillResults } from "./skill/generic-results";
+import { getDurability as getDurability } from "./utils";
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)